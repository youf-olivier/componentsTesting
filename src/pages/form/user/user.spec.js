import React from 'react';
import { render } from '@testing-library/react';
import Component from './user';

describe('User card test suite', () => {
  const user = {
    login: '123123',
    avatar_url: 'http://avatar.jpg',
    html_url: 'http://comptegithub.com',
  };

  it('renders correctly', () => {
    const { asFragment } = render(<Component user={user} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
