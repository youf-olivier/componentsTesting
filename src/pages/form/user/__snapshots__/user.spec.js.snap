// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User card test suite renders correctly 1`] = `
<DocumentFragment>
  <div
    class="user-card"
    data-testid="usercard"
  >
    <a
      class="user-card__link"
      href="http://comptegithub.com"
    >
      <img
        alt="123123"
        class="user-card__avatar"
        src="http://avatar.jpg"
      />
      <span
        class="user-card__login"
      >
        123123
      </span>
    </a>
  </div>
</DocumentFragment>
`;
